File: src/assets/usage-examples-code/utilities/delay/delay-1-basic-usage.cs
using System;
using SplashKitSDK;

Console.WriteLine("Start delay...");

int milliseconds = 2000; // Delay for 2000 milliseconds (2 seconds)
SplashKitSDK.SplashKit.Delay(milliseconds);

Console.WriteLine("End delay after " + milliseconds + " milliseconds.");

File: src/assets/usage-examples-code/utilities/delay/delay-1-basic-usage.cpp
#include "splashkit.h"
#include <string>  // Include the string header

int main()
{
    write_line("Start delay...");
    
    int milliseconds = 2000; // Delay for 2000 milliseconds (2 seconds)
    delay(milliseconds);

    write_line("End delay after " + std::to_string(milliseconds) + " milliseconds.");

    return 0;
}
